sklearn/tests/test_calibration.py

```diff
@@ -401,6 +413,44 @@ def test_sigmoid_calibration():
         _SigmoidCalibration().fit(np.vstack((exF, exF)), exY)
 
 
+def test_temperature_scaling(data):
+    """Check temperature scaling calibration"""
+    X, y = data
+    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
+
+    clfs = [
+        make_pipeline(StandardScaler(), LinearSVC(random_state=42)),
+        LogisticRegression(),
+    ]
```

Comment:
Let's pass this as an argument of the test using `@pytest.mark.parametrize` instead.